docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -36,11 +36,10 @@ components:
       required: true
       schema:
         description: >
-          Either an 8-character alphanumeric unique identifier generated by the service
-          or an authority-provided identifier
+          Either a unique identifier generated by the service or an authority-provided identifier
         oneOf:
           - type: string
-            description: A short string: the group ID assigned by the service
```

Comment:
Whoops. A "fix" I made in a previous commit actually made the document invalid (the colon without escaping).